notifications_utils/field.py

```diff
@@ -49,6 +49,17 @@ def __repr__(self):
 
 
 class Field:
+
+    """
+    An instance of Field represents a string of text containing
```

Comment:
This is good. Might be worth clarifying that the placeholders may or may not be present e.g.

An instance of Field represents a string of text, which may or may not contain placeholders which can be replaced with values.

Reply:
Updated and squashed